src/MeetingsSDKAdapter.js

```diff
@@ -205,10 +213,10 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
         })),
       )
       : of({permission: null, stream: null});
-    const video$ = mediaSettings.sendVideo
+
+    return mediaSettings.sendVideo
```

Comment:
if `sendVideo` is false, the audio messages are not emitted even if `sendAudio` is true

The mapping needs to be done on the `{permission: null, stream: null}` messages as well:

```js
const getStreamWithPermission = (condition, ID, mediaDirection, audioVideo) =>
  condition
    ? this.getStream(ID, mediaDirection, audioVideo)
    :  of({permission: null, stream: null});

const {sendAudio, sendVideo} = mediaSettings;

return getStreamWithPermission(sendAudio, ID, {sendAudio: true}).pipe(
  map(...),
  chainWith((audio) => getStreamWithPermission(sendVideo, ID, {sendVideo: true}).pipe(
    map(...),
  ),
);